plugins/harddisk/src/main/java/assets/js/harddisk.js

```diff
@@ -283,6 +283,7 @@ function harddisk(moduleId) {
             }
 
             html.push('<td data-name="', value.name, '">', icon, '&nbsp;', value.name, '</td>');
+            html.push('<td data-name="', value.size, '" class="size">', value.size, '</td>');
```

Comment:
The data-name attribute isn't necessary or at least not until you do the directory size calculation and it shouldn't the value.name or even the full path of the file.